source/less/blocks/product-item.less

```
@@ -1,12 +1,121 @@
+.product-item {
+  border-top: 1px solid @special-grey;
+  box-sizing: border-box;
+
+  &:last-child {
+    border-bottom: 1px solid @special-grey;
+  }
+
+  @media (min-width: @tablet-width) {
+    margin: 0;
+  }
+
+  @media (min-width: @desktop-width) {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: nowrap;
+    align-items: center;
+  }
+}
+
+.product-item__info {
+  @media (min-width: @tablet-width) {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    box-sizing: content-box;
```

```suggestion
    box-sizing: border-box;
```
Вот тут можно еще почитать, что content-box - это итак умолчательное значение, поэтому его не надо писать дополнительно.
http://htmlbook.ru/css/box-sizing

Нужен, обычно, `border-box`